### ISSUE TITLE
[Model]  LightGBM churn prediction model

### ISSUE THREAD

**Issue by nobrain711** [SKN26-2nd-1st/2nd_project#40]
### LightGBM (LGBMClassifier)

Credit Card Churn Dataset을 기반으로 고객 이탈 여부를 예측하는
Binary Classification 모델을 학습합니다.

---

## ⚙️ Training Setup

- Dataset  
  - Credit Card Customers (Churn Dataset)

- Feature  
  - 고객 거래 패턴
  - 카드 사용 금액
  - 거래 횟수
  - 고객 활동 관련 파생 Feature

- Hyperparameter (initial)
```python
LGBMClassifier(
    n_estimators=300,
    learning_rate=0.05,
    max_depth=6,
    num_leaves=31,
    random_state=42
)
```

---

## 📈 Expected Improvement

Baseline 모델(Logistic Regression / RandomForest 등) 대비
성능 향상을 목표로 합니다.

Target Metrics

- Accuracy: **≥ 0.80**
- Recall: **≥ 0.70**
- F1 Score: **≥ 0.78**
- ROC-AUC: **≥ 0.85**
- PR-AUC: **≥ 0.80**

Churn prediction 문제 특성상
Accuracy뿐 아니라 F1 Score와 PR-AUC를 주요 평가 지표로 사용합니다.
